Multisig/UI/Assets/AssetsViewController.swift

```diff
@@ -50,7 +50,16 @@ class AssetsViewController: ContainerViewController {
             object: nil)
         
         NotificationCenter.default.addObserver(
-            self, selector: #selector(selectedSafeUpdatedReceived), name: .selectedSafeUpdated, object: nil)
+            self,
+            selector: #selector(selectedSafeUpdatedReceived),
+            name: .selectedSafeUpdated,
+            object: nil)
+        
+        NotificationCenter.default.addObserver(
+            self,
+            selector: #selector(selectionChanged),
+            name: .selectedSafeChanged,
```

Comment:
Are selectionChanged and selectedSafeUpdatedReceived notifications referring to same event ? can we maybe merge them?

Reply:
they're different events